visualstudio-installer/tools/ChocolateyUninstall.ps1

```diff
@@ -1,3 +1,3 @@
 ﻿Uninstall-VisualStudio `
     -PackageName 'visualstudio-installer' `
-    -InstallerTechnology 'WillowVS2017OrLater
+    -InstallerTechnology 'WillowVS2017OrLater' `
```

Comment:
The backtick (` = line continuation mark) is unnecessary, since this is the last argument passed to the cmdlet.

Reply:
aaahh thanks :). but I've never tested without backtick right in this case

Reply:
I'm testing it right now :)